src/benchmarks/micro/corefx/System.Collections/Sort.cs

```diff
@@ -11,36 +11,48 @@
 namespace System.Collections
 {
     [BenchmarkCategory(Categories.CoreCLR, Categories.Collections, Categories.GenericCollections)]
-    [GenericTypeArguments(typeof(int))] // value type
-    [GenericTypeArguments(typeof(string))] // reference type
+    [GenericTypeArguments(typeof(int))] // value type, Array sort in native code
```

Comment:
@adamsitnik `string` is the reason the benchmark take and insubordinate amount of time e.g. 15 minutes with 50000 iterations. There is no big value in keeping it in my mind, at 5000 it will still be slow. We are taking magnitudes slower, hence it dominates the total benchmark run, at the cost of precision for the other tests.

However, I have approved your suggestion, then you can decide if you still want to keep it :)